recipes/libpng/all/conanfile.py

```diff
@@ -37,6 +38,7 @@ class LibpngConan(ConanFile):
         "sse": True,
         "vsx": True,
         "api_prefix": "",
+        "executables": True,
```

Comment:
this option doesn't really make sense since these utilities are removed in package() anyway.

Reply:
should `PNG_EXECUTABLES=OFF` be set unconditionally then?

Reply:
yes

Reply:
thanks, adjusted